packages/gatsby/src/utils/webpack.config.js

```diff
@@ -350,6 +356,7 @@ module.exports = async (
 
     return {
       modules: [...root, path.join(__dirname, `../loaders`), `node_modules`],
+      plugins: [PnpWebpackPlugin.moduleLoader(`${directory}/`)],
```

Comment:
Not sure what this does, add a comment maybe

Reply:
One problem/quirk with how Webpack treats loaders and plugins is that when it tries to resolve a loader, the "issuer" (the file that makes the require call) is set to the file that makes the require to the imported file (rather than the file that defines the loader - which would arguably be hard to figure out). It means that if a file from a third-party package requires another file that would go through a loader, the call will fail (because the third-party package wouldn't list the loader in its dependencies).

This line simply instructs Webpack that the non-absolute loaders should all be resolved relative to the directory of the application, regardless of who their issue is. It's not a problem for third-party-provided loaders (such as the ones added by Gatsby) because they use `require.resolve` anyway, which shortcuts this resolution 🙂